src/py/crankshaft/crankshaft/clustering/kmeans.py

```diff
@@ -0,0 +1,17 @@
+from sklearn.cluster import KMeans
+import plpy
+
+def kmeans(query, no_clusters, no_init=20):
+    data = plpy.execute('''select array_agg(cartodb_id order by cartodb_id) as ids,
```

Comment:
Yeah right now its restricted to geo. No reason we couldn't use other variables as well but for this example we only needed to do it in the geo space. I think we should run with this now and perhaps write another more general function to do k-means over geo + variables. We will need to think about that a little more cause we need to assure that each variable is scaled appropriately